andela/ah-frontend-invictus
# #166492273 Read Articles

#### What does this PR do?
This PR adds the read articles feature. A user can either signup or login and read a particular article. The user will be welcomed with a landing page that has a list of articles. When they click on one, they are then redirected to the article page to read the article.

#### Description of Task to be completed
- Retrieve all articles

#### How should this be manually tested?
1. clone the repo
2. run `npm install`
3. run `npm start`
4. login on `http://localhost:3000/login`
5. see all articles here `http://localhost:3000/`
6. read an article using this link `http://localhost:3000/articles/<article-id>/`

#### Any background context you want to provide?
The HTML templating and CSS styling is not yet done.

#### What are the relevant pivotal tracker stories?
[#166492273](https://www.pivotaltracker.com/story/show/166492273)

#### Screenshots (if appropriate)
<img width="1439" alt="Screen Shot 2019-06-16 at 14 15 42" src="https://user-images.githubusercontent.com/14992011/59563344-6d262080-9041-11e9-964b-70d8def5bc62.png">
